include/ajax.users.php

```diff
@@ -61,13 +65,16 @@ function search($type = null) {
                 return $this->search($type);
             }
 
+            // Omit already-imported remote users
             if ($emails = array_filter($emails)) {
-                $users->chain(User::objects()->filter(array(
```

Comment:
`chain` method doesn't exist. It's called `union`. That's what the patch changes